vpr/src/route/clock_connection_builders.cpp

```diff
@@ -92,18 +92,18 @@ void RoutingToClockConnection::create_switches(const ClockRRGraphBuilder& clock_
 RRNodeId RoutingToClockConnection::create_virtual_clock_network_sink_node(int x, int y) {
     auto& device_ctx = g_vpr_ctx.mutable_device();
     auto& rr_graph = device_ctx.rr_nodes;
-    auto& temp_rr_graph = device_ctx.rr_graph; //  replace this with rr_graph once device_ctx.rr_nodes is unused
+    //auto& temp_rr_graph = device_ctx.rr_graph; //  replace this with rr_graph once device_ctx.rr_nodes is unused
     auto& rr_graph_builder = device_ctx.rr_graph_builder;
     auto& node_lookup = device_ctx.rr_graph_builder.node_lookup();
     rr_graph.emplace_back();
```

Comment:
@tangxifan 
I am trying to replace with rr_graph_view in line 98. How Can I replace here?